### PR TITLE
Minor improvements on AstelcoDome. 

### PR DESCRIPTION
**Pull request by tribeiro**
Add move to azimuth, to move the dome when in stand mode. It also fixes indentation to comply with PEP8.

### CODE REVIEW

File: src/chimera/instruments/astelcodome.py [hunk 1]
```diff
@@ -34,181 +33,214 @@
...
-    __config__ = {	'user'			: 'admin',
-					'password'		: 'admin',
-					'ahost'			: 'localhost',
-                    'aport'			: "65432" }
+    __config__ = {'user': 'admin',
+                  'password': 'admin',
+                  'ahost': 'localhost',
+                  'aport': "65432"}
+                  "maxidletime": 90.,
+                  "stabilization_time": 5.}
```
Comment (wschoenell) [inline src/chimera/instruments/astelcodome.py]: This config statement is wrong. You finish the dict, but put 2 lines more.

File: src/chimera/instruments/astelcodome.py [hunk 2]
```diff
@@ -231,8 +263,6 @@ def open(self):
             print self._tpl.getobject('SERVER.UPTIME')
             self._tpl
```
Comment (wschoenell) [inline src/chimera/instruments/astelcodome.py]: L260, 262 and 264 are useless.

File: src/chimera/instruments/astelcodome.py [hunk 3]
```diff
@@ -240,101 +270,92 @@ def open(self):
...
 
 
     def isSlitOpen(self):
-		self._slitPos = self._tpl.getobject('AUXILIARY.DOME.REALPOS')
-		self._slitOpen = self._slitPos > 0
-		#self.log.debug(self._tpl.getobject('AUXILIARY.DOME.OPEN_MASK'))
-		return self._slitOpen
+        self._slitPos = self._tpl.getobject('AUXILIARY.DOME.REALPOS')
+        self._slitOpen = self._slitPos > 0
+        #self.log.debug(self._tpl.getobject('AUXILIARY.DOME.OPEN_MASK'))
```
Comment (wschoenell) [inline src/chimera/instruments/astelcodome.py]: Remove these unused (and commented) lines.

File: src/chimera/instruments/astelcodome.py [hunk 4]
```diff
@@ -34,181 +33,214 @@
 from chimera.core.exceptions import ChimeraException
```
Comment (wschoenell) [inline src/chimera/instruments/astelcodome.py]: Unused import InvalidDomePositionException on L28.

**Comment by wschoenell** [comment]
@tribeiro,

Thanks for doing the suggested changes. Since that this works with our ASTELCO telescope in Chile and this PR only affects the astelcodome.py. I suggest to merge it.

**Comment by phsilva** [comment]
Looks good to me.

**Comment by wschoenell** [comment]
Merged #52.

**Comment by ankanaan** [comment]
http://www.google.com.br/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0CB4QtwIwAA&url=http%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DBovQyphS8kA&ei=NglIVbW-G-3gsATi7YG4DQ&usg=AFQjCNHmQpGCkBIwLBq01ag05vMhcD_GFw&sig2=ZGAkTMEYtQkz5IAUZQIeOA&bvm=bv.92291466,d.cWc

**Pull request merged**